Assets/Scripts/PlayerCharacter/PlayerGun.cs

```diff
@@ -0,0 +1,116 @@
+using UnityEngine;
+using UnityEngine.InputSystem;
+
+namespace Cosmobot
+{
+    public class PlayerGun : MonoBehaviour, DefaultInputActions.IPlayerGunActions
+    {
+
+        [SerializeField] private PlayerCamera playerCamera;
+        [SerializeField] private Transform cameraTransform;
+        [SerializeField] private float maxPickupRange; // relative to character, not camera
```

Comment:
This comment can be set as tooltip

```suggestion
        [SerializeField, Tooltip("relative to character, not camera")] 
        private float maxPickupRange;
```